crates/red_knot/src/module.rs

```diff
@@ -225,22 +225,23 @@ struct ModuleSearchPathInner {
     kind: ModuleSearchPathKind,
 }
 
-#[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Hash, is_macro::Is)]
 pub enum ModuleSearchPathKind {
```

Comment:
Worth a comment that these are listed here in priority order? And even a link to the relevant page of the typing spec (soon hopefully to be updated to match what we're doing here)?